db/projects/ethereum-classic/project.yaml

```diff
@@ -0,0 +1,76 @@
+name: Ethereum Classic
+web:
+  - 'https://ethereumclassic.org/'
+start_date: '2016-07-23'
```

Comment:
This point I am leaving still opened. I have changed the date, but to 2015-07-30, when ETH was released. I think this should be correct start_date of the project (start_date of the ledger should be the for date), since the project is only continuing of ETH project (and also discutabital, but some says the ETC is the true ETH :) ) Anyway, your comments are welcomed here.

Reply:
Very good point 👍. I have thought about it a bit, and i think we can compare it with database of countries, for example:
- `Project` is like a Country (eq. Czech Republic, Czechoslovakia etc.)
- `Ledger ` is like a Constitution or People inside of country
- `Fork_date` is like a Revolution day - change from one country to another

Czech Republic & Czechoslovakia haved almost same constitution & people - like ETH & ETC ledger, but you cannot say that Czech Republic (ETC) was Czechoslovakia (ETH). Nobody know about Czech Republic before 100 years, same like nobody know ETC when ETH was started.
So, about ETC - i think right way is this:
- `project start_date` = `ledger fork date` = 2016-07-20..
- `ledger start_date` = ETH start date = 2015-07-30

Reply:
Whats @coinmandeer opinion?

Reply:
Changed after the discussion on Hackaton